packages/0x.js/src/utils/order_validation_utils.ts

```diff
@@ -124,31 +124,12 @@ export class OrderValidationUtils {
         if (!_.isUndefined(expectedFillTakerTokenAmount)) {
             fillTakerTokenAmount = expectedFillTakerTokenAmount;
         }
-        const fillMakerTokenAmount = OrderValidationUtils._getPartialAmount(
+        await OrderValidationUtils.validateFillOrderBalancesAllowancesThrowIfInvalidAsync(
```

Comment:
How does this work without throwing an error? `validateFillOrderBalancesAllowancesThrowIfInvalidAsync` also attempts a `Fee` transfer from taker `0x0`...